shopfloor_example/services/partner_service.py

```diff
@@ -15,7 +15,9 @@ class PartnerExampleService(Component):
     _description = __doc__
 
     @restapi.method(
-        [(["/scan/<string:identifier>"], "GET")], auth="api_key",
+        [(["/scan/<string:identifier>"], "GET")],
+        output_param=restapi.CerberusValidator("scan"),
```

Comment:
that's why they were working: by default lookup to _validate_*.
I  guess `auth` is not required anymore tnx to @acsonefho fix, right?

Reply:
by default, not validated at all....
We have to test id auth is no more required...